frontend/src/components/WalletMenu.vue

```diff
@@ -101,19 +112,28 @@ const walletOptions = ref([
 ]);
 
 onMounted(() => {
-  if (isMobile(window.navigator.userAgent)) {
-    const metaMaskAvailable = window.ethereum && window.ethereum.isMetaMask;
-    const coinbaseAvailable = window.ethereum && window.ethereum.isCoinbaseWallet;
+  const metaMaskAvailable = window.ethereum && window.ethereum.isMetaMask;
+  const coinbaseAvailable = window.ethereum && window.ethereum.isCoinbaseWallet;
 
+  if (isMobile(window.navigator.userAgent)) {
     if (metaMaskAvailable) {
       walletOptions.value = walletOptions.value.filter((option) => option.name === 'MetaMask');
     } else if (coinbaseAvailable) {
       walletOptions.value = walletOptions.value.filter((option) => option.name === 'Coinbase');
     } else {
       walletOptions.value = walletOptions.value.filter((option) => option.hasMobileFlow);
     }
+  } else {
+    if (metaMaskAvailable) {
+      walletOptions.value = walletOptions.value.filter(
+        (option) => option.name !== 'Browser Wallet',
+      );
+    } else {
+      walletOptions.value = walletOptions.value.filter((option) => option.name !== 'MetaMask');
+    }
```

Comment:
You are right. Nice catch!
I will just show MetaMask as an option when there is no injection detected since we only have onboarding support for MM currently.